sources/editor/Stride.GameStudio/ViewModels/DebuggingViewModel.cs

```diff
@@ -162,50 +162,56 @@ private async void PullAssemblyChanges([NotNull] ProjectWatcher projectWatcher)
             var changesBuffer = new BufferBlock<AssemblyChangedEvent>();
             using (projectWatcher.AssemblyChangedBroadcast.LinkTo(changesBuffer))
             {
-                while (!assemblyTrackingCancellation.IsCancellationRequested)
+                while (true)
```

Comment:
Is an infinite loop really needed? There should be way to exit the loop gracefully.

Reply:
https://github.com/stride3d/stride/pull/2106/commits/1620c639555d9b6a9ef32338ad562db7007a86d8
i changed it back to the previous cancelation way, as it didnt change in the logic there